index.js

```diff
@@ -41,6 +41,44 @@ function onReady () {
         client.write(consts.targetIds.app, consts.eventNames.displayEventRemoved, {displays: {all: screen.getAllDisplays(), primary: screen.getPrimaryDisplay()}})
     })
 
+    const powerMonitor = electron.powerMonitor
+    // Listen to power events
+    powerMonitor.on('suspend', function() {
+        client.write(consts.targetIds.power, consts.eventNames.powerEventSuspend)
```

Comment:
As mentioned in the go PR, could you remove the power target id and use the app's one instead?

Reply:
Done